src/org/zaproxy/zap/extension/pscanrules/CookieSecureFlagScanner.java

```diff
@@ -72,29 +76,53 @@ public void scanHttpResponseReceive(HttpMessage msg, int id, Source source) {
 			iterator.addIterator(cookies2.iterator());
 		}
 
+		List<String> ignoreList = new ArrayList<String>();
+		String ignoreConf = getModel().getOptionsParam().getConfig().
```

Comment:
Since you added a `setModel()` function, would it be better to call that in `init()` and then use `model` here?